doc/playbook.rst

```diff
@@ -120,6 +128,8 @@ Please note that the tag does not include a ``v`` at the beginning.
    In practice, almost any Science Pipelines update is at least a minor version, because new features are added constantly.
    In the future, there may be "patched weekly" builds, which would justify a patch version of Prompt Processing.
 
+   **Guidance on designating major releases:**
+
```

Comment:
I think it would make sense to include this information on a different page and link it. The thought with the bolded headlines was really just to give the reader a signal that this section is not relevant unless one of the following bullet point conditions is met. For now we could make a header with something like "The following cases will always trigger a major release:" instead of "Guidance on". I was thinking more like official rules for something but I definitely wouldn't want it to sound optional. For now I'll change the header and keep it as is, but we can move the entire subsection out in future if it makes more sense to do so.